validator/client/registration.go

```diff
@@ -30,6 +32,10 @@ func SubmitValidatorRegistrations(
 	if _, err := validatorClient.SubmitValidatorRegistrations(ctx, &ethpb.SignedValidatorRegistrationsV1{
 		Messages: signedRegs,
 	}); err != nil {
+		if strings.Contains(err.Error(), builder.ErrNoBuilder.Error()) {
+			log.Warnln("Beacon node does not utilize a custom builder. Validator registration skipped.")
```

Comment:
Is there anything you can add here to give information to the user on how to correct the issue? I.e. "Restart your beacon node with the --mev-relay flag" or something like that

Reply:
+1, I think more verbosity will be nice here such as validator keys and such

Reply:
I added the flag name to the log. It impacts all validator keys, so there's no need to add any.